### ISSUE TITLE
Update filenames for config and database

### ISSUE THREAD

**Issue by jrozner** [ashirt-ops/ashirt#33]
Change the old filenames from `screenshot.json` and `screenshot.sqlite` to `config.json` and `evidence.sqlite`. Might as well do this ahead of tagging a 1.0.

**Comment by jrozner** [comment]
The follow will need to be run as a manual upgrade step:
```sh
mv ~/Library/Preferences/ashirt/screenshot.json ~/Library/Preferences/ashirt/config.json
mv ~/Library/Preferences/ashirt/screenshot.sqlite ~/Library/Preferences/ashirt/evidence.sqlite
```

**Comment by jrozner** [comment]
This was merged as part of #39. The following manual changes need to be made:

```
mv ~/Library/Preferences/ashirt/screenshot.json "$HOME/Library/Application Support/ashirt/config.json"
mv ~/Library/Preferences/ashirt/screenshot.sqlite "$HOME/Library/Application Support/ashirt/evidence.sqlite"
```

We also are by default moving the storage of evidence to `$HOME/Library/Application Support/ashirt/evidence`. Not change is necessary here but is an option in the migration.
